README.md

```diff
@@ -1,4 +1,4 @@
 # Deploy portal
 [![CircleCI build status](https://circleci.com/gh/canonical-web-and-design/webteam.canonical.com.svg?style=shield)](https://circleci.com/gh/canonical-web-and-design/webteam.canonical.com) [![Code coverage](https://codecov.io/gh/canonical-web-and-design/webteam.canonical.com/branch/main/graph/badge.svg)](https://codecov.io/gh/canonical-web-and-design/webteam.canonical.com)
```

Comment:
would you mind removing those labels too. We don't use them anymore.

Reply:
Done